source/sass/main.scss

```diff
@@ -327,23 +328,25 @@ button {
     .container {
       margin-top: 0;
       margin-bottom: 0;
-      min-height: 220px;
-
-      @media screen and (min-width: $bp-md) {
-        display: flex;
-        justify-content: space-around;
-        align-items: center;
-      }
     }
 
     .info-box {
-      margin: 1.5rem 0;
-      display: inline-block;
-      width: 100%;
-      max-width: 342px;
-      font-size: 24px;
-      font-weight: 500;
-      line-height: 33px;
+      margin-top: 1.5rem;
+
+      &:not(:first-child) {
```

Comment:
I just tried too, and it's not compatible with the `.row` parent. We would need to override row, which I don't think is worth it.

It's OK, I'm not actually worried. I just wanted to understand the behaviour of collapsing margins and bootstrap better.

If you're curious, the flexbox setting of `.row` cancels the collapsing margins. I'm not 100% sure why, but my guess is flexbox would be used to layout things via `space-around` and `space-between`, and not margins? *shrugs*